cmd/sync.go

```diff
@@ -64,6 +64,9 @@ type rawSyncCmdArgs struct {
 	// otherwise the user is prompted to make a decision
 	deleteDestination string
 
+	// this flag is to disable comparator and overwrite files at destination irrespective
+	mirrorMode bool
```

Comment:
I'll happily change this name to something else.

Reply:
Mirror mode - as far as this change goes- does copy alone, and you'll have to additionally use delete-destination with --mirrorMode to achieve what we think is intuitional.